### DISCUSSION TITLE
How to change the number of posts on a page?

### DISCUSSION THREAD

**Discussion by tonylitianyu** [adityatelange/hugo-PaperMod#320] [Q&A] [answered]
By default, 5 posts are on the home page. Is there a way to show all the posts at once? Thanks!

**Comment by kdkasad** [comment, marked as answer]
This is explained in the Hugo wiki:
https://gohugo.io/templates/pagination/#configure-pagination

**Reply by tonylitianyu** [reply]
Thank you! I am very new to Hugo.

**Reply by kdkasad** [reply]
No problem! A lot of things aren't theme-specific. I you're having a problem (or just a question) that isn't necessarily related to the theme, check the Hugo docs, as it might be explained there.
